psychopy/visual/window.py

```diff
@@ -1102,8 +1102,12 @@ def close(self):
             openWindows.remove(self)
         except Exception:
             pass
-        if not self.useNativeGamma and self.origGammaRamp is not None:
-            self.backend.gammaRamp = self.origGammaRamp
```

Comment:
This change means that we're always using this mechanism whereas, before, we were using pygame's own gamma correction for pygame windows and only using our function for pyglet. Then again, I guess it will work everywhere and we might have more confidence in our own function than one in another external API. So I guess this is fine, but I'm just noting that it's a change

Reply:
I think this is a side effect of being able to restore the system gamma on window closure when the gamma is set after window creation. I had wanted to stop the check on ```useNativeGamma``` (because that is not updated if gamma is set after window creation), but I hadn't picked up on the ```backend``` change.

I think your way of using the backend to restore the gamma is better - presumably the definition of ```self.origGammaRamp``` should use it too.

Reply:
New issue in #1618 might be relevant - perhaps pygame windows should also use the custom gamma methods.